# Joint angles of the robot

float64 theta_1 # Yellow joint
float64 theta_2 # Blue joint (#1)
float64 theta_3 # Blue joint (#2)
float64 theta_4 # Green joint
